cmake/VerifyCompiler.cmake

```diff
@@ -47,15 +47,15 @@ if(HIP_PLATFORM STREQUAL "hcc")
 
     # Workaround until hcc & hip cmake modules fixes symlink logic in their config files.
     # (Thanks to rocBLAS devs for finding workaround for this problem.)
-    list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hcc /opt/rocm/hip)
```

Comment:
ROCM_PATH? We should not use CMAKE_INSTALL_PREFIX

Reply:
I can remove CMAKE_INSTALL_PREFIX and use ROCM_PATH, but it is set only during the CI build and no default value set to /opt/rocm. Then I have to update the changes to keep both the paths. ${ROCM_PATH}/hcc and /opt/rocm/hcc